lib/trino-orc/src/main/java/io/trino/orc/OrcReader.java

```diff
@@ -65,7 +65,7 @@
 
 public class OrcReader
 {
-    public static final int MAX_BATCH_SIZE = 1024;
+    public static final int MAX_BATCH_SIZE = 8048;
```

Comment:
why not 8192 ?

Reply:
you are right :). It shouldn't affect benchmark results

Reply:
Can we have any benchmark numbers post this increase ?

Reply:
Yes. I have to rerun benchmarks due to some flakyness